### PR TITLE
chore: use logforth to replace fern

### PR DESCRIPTION
**Pull request by andylokandy**
I hereby agree to the terms of the CLA available at: https://docs.databend.com/dev/policies/cla/

## Tests

- [x] Unit Test
- [ ] Logic Test
- [ ] Benchmark Test
- [ ] No Test - _Explain why_

## Type of change

- [ ] Bug Fix (non-breaking change which fixes an issue)
- [ ] New Feature (non-breaking change which adds functionality)
- [ ] Breaking Change (fix or feature that could cause existing functionality not to work as expected)
- [ ] Documentation Update
- [x] Refactoring
- [ ] Performance Improvement
- [ ] Other (please describe):

- - -
This change is [Reviewable](https://reviewable.io/reviews/datafuselabs/databend/16249)

### CODE REVIEW

**Review by Xuanwo — APPROVED** [review]
Thanks!

**Review by everpcpc — APPROVED** [review]

**Comment by sundy-li** [comment]
There are extra error messages:

```
+ (while in processor thread 0)
```

Need fix the ci tests.

File: src/common/tracing/src/loggers.rs [hunk 1]
```diff
@@ -12,333 +12,37 @@
...
-    match ThreadTracker::query_id() {
-        None => {
-            out.finish(format_args!(
-                r#"{{"timestamp":"{}","level":"{}","fields":{}}}"#,
-                chrono::Local::now().to_rfc3339_opts(chrono::SecondsFormat::Micros, true),
-                record.level(),
-                serde_json::to_string(&fields).unwrap_or_default(),
-            ));
-        }
-        Some(query_id) => {
```
Comment (everpcpc) [inline src/common/tracing/src/loggers.rs:239]: `query_id` missing for the new system logging cc @andylokandy

**Comment by BohuTANG** [comment]
This PR introduces a **breaking change** to the design. Cloud query history and system logs are all broken now.

**Pull request merged**